docs/detections/prebuilt-rules/rule-details/azure-application-credential-modification.asciidoc

```diff
@@ -33,13 +33,15 @@ Identifies when a new credential is added to an application in Azure. An applica
 * SecOps
 * Identity and Access
 
-*Version*: 1
+*Version*: 3 (<<azure-application-credential-modification-history, version history>>)
```

Comment:
looks like these versions are getting bumped by 2 instead of 1

Reply:
I found the source of the double bump. Every endpoint rule had `event.ingested` added in 7.11.2 [here](https://github.com/elastic/kibana/commit/df46bfcefdcfbba49e0c4842c3f6fc88822163df#diff-1d0a7eb78fc95721f1fa7f767d89590db57b4a041792076d1b68106cb8e5afab). 

So I will need to re-run all of this to include 7.11.2 and _then_ 7.12 to include those interim changes